docs/subnets/deploying-subnets-on-prod.md

```diff
@@ -0,0 +1,189 @@
+
+You can also use Subnet-CLI to manage your Subnet after the initial deploy, to add new validators, renew ones that expired and so on. Familiarize yourself with Subnet-CLI, you be using it often!
+
+## Validator Configuration
+
+Running nodes as Subnet validators warrants some additional considerations, above those when running a regular node or a Primary Network-only validator.
+
+### Subnet Whitelisting
+
+For a node to join a subnet, there are two prerequisites:
+* Primary Network validation
+* Subnet whitelisting
+
+Primary Network validation means that a node cannot join a subnet as a validator before becoming a validator on the Primary Network itself. So, after you add the node to the validator set on the Primary Network, node can join a subnet. Of course, this is valid only for subnet validators, if you need a non-validating subnet node, then the node doesn't need to be a validator at all.
+
+To have a node start syncing the Subnet, you need to add the `--whitelisted-subnets` command line option, or `whitelisted-subnets` key to the node config file (found at `.avalanchego/configs/node.json` for installer-script created nodes). A single node can sync multiple subnets, so you can add them as a comma-separated list of subnet IDs.
+
+An example of a node config syncing two subnets:
+
+```json
+{
+  "dynamic-public-ip": "opendns",
+  "http-host": "",
+  "whitelisted-subnets": "28nrH5T2BMvNrWecFcV3mfccjs6axM1TVyqe79MCv2Mhs8kxiY,Ai42MkKqk8yjXFCpoHXw7rdTWSHiKEMqh5h8gbxwjgkCUfkrk"
+}
+```
+
+But that is not all. Besides the whitelist containing the SubnetID, node also needs to have the plugin that contains the VM instance the blockchain in the Subnet will run. You should have already been through that on testnet and Fuji, but for a refresher, you can refer to [this tutorial](create-a-fuji-subnet.md).
+
+So, name the VM plugin binary as the `VMID` of the Subnet chain and place it in the `plugins` directory where the node binary is (for installer-script created nodes that would be `~/avalanche-node/plugins/`). 
+
+### Subnet Bootstrapping
+
+After you have added the whitelist and placed the VM binary in the correct directory, your node is ready to start syncing with the Subnet. Restart the node and monitor the log output. You should notice something similar to:
+
+```text
+Jul 30 18:26:31 node-fuji avalanchego[1728308]: [07-30|18:26:31.422] INFO chains/manager.go:262 creating chain:
+Jul 30 18:26:31 node-fuji avalanchego[1728308]:     ID: 2ebCneCbwthjQ1rYT41nhd7M76Hc6YmosMAQrTFhBq8qeqh6tt
+Jul 30 18:26:31 node-fuji avalanchego[1728308]:     VMID:srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy
+```
+
+That means the node has detected the subnet, and is attempting to initialize it and start bootstrapping the Subnet. It might take some time (if there are already transactions on the subnet), and eventually it will finish the bootstrap with a message like:
```

Comment:
```suggestion
That means the node has detected the subnet, and is attempting to initialize it and start bootstrapping the Subnet. It might take some time (if there are already transactions on the Subnet), and eventually it will finish the bootstrap with a message like:
```